src/app/work-item/work-item.component.ts

```diff
@@ -60,7 +60,7 @@ export class WorkItemComponent implements OnInit, AfterViewInit {
     if (this.loggedIn) {
       this.filters.push({
         id:  1,
-        name: 'Assign to Me',
+        name: 'Assign to me',
```

Comment:
Your commit message is Rename filter method "Assign to Me" to "Assigned to me" but you rename it "Assign to Me" to "Assign to me"